Rakefile

```diff
@@ -121,31 +112,25 @@ namespace :test do
       'test/functional/**/*_test.rb',
       'lib/plugins/inspec-*/test/functional/**/*_test.rb',
     ])
-    t.warning = false # This just complains about things in underlying libraries
+    t.warning = true
     t.verbose = true
     t.ruby_opts = ['--dev'] if defined?(JRUBY_VERSION)
   end
   # Inject a prerequisite task
   task :functional => [:accept_license]
 
-  # Functional tests on Windows take a bit to run. This
-  # optionally takes a env to breake the tests up into 3 workers.
-  Rake::TestTask.new(:'functional:windows') do |t, args|
-    files = Dir.glob('test/functional/*_test.rb').sort
-    if ENV['WORKER_NUMBER']
-      count = (files.count / 3).abs+1
-      start = (ENV['WORKER_NUMBER'].to_i - 1) * count
-      files = files[start..start+count-1]
-    end
-
+  Rake::TestTask.new(:unit) do |t|
     t.libs << 'test'
-    t.test_files = files
-    t.warning = false # This just complains about things in underlying libraries
+    t.test_files = Dir.glob([
+      'test/unit/**/*_test.rb',
+      'lib/plugins/inspec-*/test/unit/**/*_test.rb',
+    ])
+    t.warning = true
     t.verbose = true
     t.ruby_opts = ['--dev'] if defined?(JRUBY_VERSION)
   end
   # Inject a prerequisite task
-  task :'functional:windows' => [:accept_license]
+  task :unit => [:accept_license]
```

Comment:
I'd like to get rid of `:accept_license` at some point. The tests should do what they need to do themselves.

Reply:
I agree, at the very least can we make it automatically part of all the tasks in a namespace?